src/Interpreters/TreeRewriter.cpp

```diff
@@ -121,16 +121,64 @@ struct CustomizeAggregateFunctionsSuffixData
     void visit(ASTFunction & func, ASTPtr &) const
     {
         const auto & instance = AggregateFunctionFactory::instance();
-        if (instance.isAggregateFunctionName(func.name) && !endsWith(func.name, customized_func_suffix))
+        if (instance.isAggregateFunctionName(func.name) && !endsWith(func.name, "OrNull"))
         {
             auto properties = instance.tryGetProperties(func.name);
             if (properties && !properties->returns_default_when_only_null)
-                func.name = func.name + customized_func_suffix;
+            {
+                if (!endsWith(func.name, customized_func_suffix))
```

Comment:
It seems like `customized_func_suffix = "OrNull"` here and this check is equal to this:
https://github.com/ClickHouse/ClickHouse/blob/91c3de966416f3e5a1338ca7a82293984d92a672/src/Interpreters/TreeRewriter.cpp#L124

And again, I can't understand why do we use `"OrNull"` instead of `customized_func_suffix`

Reply:
It's my mistake, all places should be `customized_func_suffix`.

Reply:
And what about double check `!endsWith(func.name, customized_func_suffix)` here?